### PR TITLE
HKEY_LOCAL_MACHINE_MACHINE typo correction

### LINKED ISSUE #9373: Typo in item 25. Windows Spotlight last bullet DisableWindowsConsumerFeatures: HKEY_LOCAL_MACHINE_MACHINE
The last bullet point in Section 25. Windows Spotlight (https://docs.microsoft.com/en-us/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services#25-windows-spotlight) references a key containing a hive that should not exist:

`HKEY_LOCAL_MACHINE_MACHINE\SOFTWARE\Policies\Microsoft\Windows\CloudContent`

This should likely be corrected to:

`HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\CloudContent`

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9abd3113-b803-87b6-ef7f-6893bcc5ceb5
* Version Independent ID: 6f7ca1ee-d19c-42bb-7db5-6158fc160e32
* Content: [Manage connections from Windows 10 operating system components to Microsoft services - Windows Privacy](https://docs.microsoft.com/en-us/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services)
* Content Source: [windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @linque1
* Microsoft Alias: **robsize**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As reported in issue ticket #9373 (**Typo in item 25. Windows Spotlight last bullet DisableWindowsConsumerFeatures: HKEY_LOCAL_MACHINE_MACHINE**), `/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md#L1769` contains a redundant extra occurrence of the key phrase `_MACHINE` (`HKEY_LOCAL_MACHINE_MACHINE`) (https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public@%7B2021-03-28T19:38:33Z%7D/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md#L1769)

Because this document is rather massive (1917 lines, 119 993 characters), I will mainly be focusing on the typo correction, to avoid losing the overview of my changes.

(Additional whitespace corrections and improvements are added, as a matter of course.)

Content: [Manage connections from Windows 10 operating system components to Microsoft services - Windows Privacy](https://docs.microsoft.com/en-us/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services)
Content Source: https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md
Product: **w10**
Technology: **windows**
GitHub Login: @linque1
Microsoft Alias: **robsize**

**Changes proposed:**
- Remove redundant occurrence of `_MACHINE` in Line 1766 (was 1769)
- Use UPPERCASE for HIVE_KEY (`HKEY_*`) names (business standard)
- Expand "regkey" to 'registry key' (1 occurrence)

**Codestyle and whitespace changes:**
- Reduce all triple blank lines (5x) to double (2 consecutive blank lines should be enough for MarkDown and HTML)
- Normalize number list and bullet point list spacing to 1 blank space in front of the text content
- Standardize MarkDown indent marker compatibility spacing, add missing blank space in Note blobs
- Normalize spacing between sentences, reduce to a single space after sentence period dots.
- Remove all redundant End-Of-Line blank spaces (140 characters)
- Add missing NewLine at End-Of-File (EOF)

**Ticket reference or closure:**

Closes #9373

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
To save yourself from the big job of scrolling through the entire document, use **Hide whitespace changes** to view only the actual effective text changes:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9374/files?diff=unified&w=1

A similar effect can be achieved when using the **Rich Diff view**, although it is more oriented towards the GitHub document Layout:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9374/files?diff=unified&w=1&short_path=22e2027

**Comment by illfated** [comment]
@JohanFreelancer9 : Please provide your copy review services as usual. Thank you.

**Comment by illfated** [comment]
#sign-off

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 1]
```diff
@@ -19,27 +19,27 @@ ms.date: 12/1/2020
...
+> [!Warning]
 > - If a user executes the **Reset this PC** command (Settings -> Update & Security -> Recovery) with the **Keep my files option** (or the **Remove Everything** option) the Windows Restricted Traffic Limited Functionality Baseline settings will need to be re-applied in order to re-restrict the device. Egress traffic may occur prior to the re-application of the Restricted Traffic Limited Functionality Baseline settings.
-> - To restrict a device effectively (first time or subsequently), it is recommended to apply the Restricted Traffic Limited Functionality Baseline settings package in offline mode. 
+> - To restrict a device effectively (first time or subsequently), it is recommended to apply the Restricted Traffic Limited Functionality Baseline settings package in offline mode.
 > - During update or upgrade of Windows, egress traffic may occur.
 
 To use Microsoft Intune cloud based device management for restricting traffic please refer to the [Manage connections from Windows 10 operating system components to Microsoft services using Microsoft Intune MDM Server](./manage-connections-from-windows-operating-system-components-to-microsoft-services-using-mdm.md)
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:45]:
```suggestion
To use Microsoft Intune cloud-based device management for restricting traffic please refer to the [Manage connections from Windows 10 operating system components to Microsoft services using Microsoft Intune MDM Server](./manage-connections-from-windows-operating-system-components-to-microsoft-services-using-mdm.md).
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 2]
```diff
@@ -1079,7 +1078,6 @@ To turn off **Let apps access my name, picture, and other account info**:
 - Create a REG_DWORD registry setting named **LetAppsAccessAccountInfo** in **HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows\\AppPrivacy** with a value of 2 (two).
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1078]:
```suggestion
- Create a REG_DWORD registry setting named **LetAppsAccessAccountInfo** in **HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows\\AppPrivacy** with a **value of 2 (two)**
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 3]
```diff
@@ -1334,7 +1332,7 @@ To turn off tailored experiences with relevant tips and recommendations by using
 
   -or-
 
-- Create a REG_DWORD registry setting named **DisableTailoredExperiencesWithDiagnosticData** in **HKEY_Current_User\\SOFTWARE\\Policies\\Microsoft\\Windows\\CloudContent** with a value of **1**
+- Create a REG_DWORD registry setting named **DisableTailoredExperiencesWithDiagnosticData** in **HKEY_CURRENT_USER\\SOFTWARE\\Policies\\Microsoft\\Windows\\CloudContent** with a value of **1**
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1335]:
```suggestion
- Create a REG_DWORD registry setting named **DisableTailoredExperiencesWithDiagnosticData** in **HKEY_CURRENT_USER\\SOFTWARE\\Policies\\Microsoft\\Windows\\CloudContent** with a **value of 1 (one)**
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 4]
```diff
@@ -1519,9 +1516,9 @@ Enterprise customers can manage their Windows activation status with volume lice
 
 - Create a REG_DWORD registry setting named **NoAcquireGT** in **HKEY_LOCAL_MACHINE\\Software\\Policies\\Microsoft\\Windows NT\\CurrentVersion\\Software Protection Platform** with a value of 1 (one).
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1517]:
```suggestion
- Create a REG_DWORD registry setting named **NoAcquireGT** in **HKEY_LOCAL_MACHINE\\Software\\Policies\\Microsoft\\Windows NT\\CurrentVersion\\Software Protection Platform** with a **value of 1 (one)**.
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 5]
```diff
@@ -1519,9 +1516,9 @@ Enterprise customers can manage their Windows activation status with volume lice
...
 
->[!NOTE]
->Due to a known issue the **Turn off KMS Client Online AVS Validation** group policy does not work as intended on Windows Server 2016, the **NoAcquireGT** value needs to be set instead.
->The Windows activation status will be valid for a rolling period of 180 days with weekly activation status checks to the KMS.
+> [!NOTE]
+> Due to a known issue the **Turn off KMS Client Online AVS Validation** group policy does not work as intended on Windows Server 2016, the **NoAcquireGT** value needs to be set instead.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1520]:
```suggestion
> Due to a known issue the **Turn off KMS Client Online AVS Validation** group policy does not work as intended on Windows Server 2016; the **NoAcquireGT** value needs to be set instead.
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 6]
```diff
@@ -1571,14 +1568,14 @@ You can disable Teredo by using Group Policy or by using the netsh.exe command.
 
 ### <a href="" id="bkmk-wifisense"></a>23. Wi-Fi Sense
 
->[!IMPORTANT]
->Beginning with Windows 10, version 1803, Wi-Fi Sense is no longer available. The following section only applies to Windows 10, version 1709 and prior. Please see [Connecting to open Wi-Fi hotspots in Windows 10](https://privacy.microsoft.com/en-us/windows-10-open-wi-fi-hotspots) for more details.
+> [!IMPORTANT]
+> Beginning with Windows 10, version 1803, Wi-Fi Sense is no longer available. The following section only applies to Windows 10, version 1709 and prior. Please see [Connecting to open Wi-Fi hotspots in Windows 10](https://privacy.microsoft.com/en-us/windows-10-open-wi-fi-hotspots) for more details.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1572]:
```suggestion
> Beginning with Windows 10, version 1803, Wi-Fi Sense is no longer available. The following section only applies to Windows 10, version 1709 and prior. Please see [Connecting to open Wi-Fi hotspots in Windows 10](https://privacy.microsoft.com/windows-10-open-wi-fi-hotspots) for more details.
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 7]
```diff
@@ -1866,35 +1863,35 @@ You can turn off Windows Update by setting the following registry entries:
...
+- Add a REG_SZ value named **WUStatusServer** to **HKEY_LOCAL_MACHINE\\Software\\Policies\\Microsoft\\Windows\\WindowsUpdate** and ensure it is blank with a space character **" "**.
 
   -and-
 
-- Add a REG_SZ value named **UpdateServiceUrlAlternate** to **HKEY_LOCAL_MACHINE\\Software\\Policies\\Microsoft\\Windows\\WindowsUpdate** and ensure it is blank with a space character **" "**. 
+- Add a REG_SZ value named **UpdateServiceUrlAlternate** to **HKEY_LOCAL_MACHINE\\Software\\Policies\\Microsoft\\Windows\\WindowsUpdate** and ensure it is blank with a space character **" "**.
 
   -and-
 
 - Add a REG_DWORD value named **UseWUServer** to **HKEY_LOCAL_MACHINE\\Software\\Policies\\Microsoft\\Windows\WindowsUpdate\\AU** and set the value to 1.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1878]:
```suggestion
- Add a REG_DWORD value named **UseWUServer** to **HKEY_LOCAL_MACHINE\\Software\\Policies\\Microsoft\\Windows\WindowsUpdate\\AU** and set the **value to 1 (one)**.
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 8]
```diff
@@ -1906,16 +1903,15 @@ For China releases of Windows 10 there is one additional Regkey to be set to pre
 
 - Add a REG_DWORD value named **HapDownloadEnabled** to **HKEY_LOCAL_MACHINE\\Software\\Microsoft\\LexiconUpdate\\loc_0804** and set the value to 0.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1904]:
```suggestion
- Add a REG_DWORD value named **HapDownloadEnabled** to **HKEY_LOCAL_MACHINE\\Software\\Microsoft\\LexiconUpdate\\loc_0804** and set the **value to 0 (zero)**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
A few additional suggestions, mainly for consistency.

**Comment by illfated** [comment]
Thank you, those looked absolutely fine. Thank you also for reviewing such a long document which I found to be somewhat tiresome and sleep-inducing on my own screen. ;-) 😉

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
@denisebmsft : Please assign someone to perform final review & merge.

**Pull request merged**